src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -873,8 +838,6 @@ public ColibriShim getColibriShim()
                 }
                 if (endpointId == null)
                 {
-                    //TODO: is it reasonable to enforce this?
-                    // If we're creating a channel, we need to know which endpoint it belongs
```

Comment:
so do you think this will be alright?  when i wrote the shims i wasn't sure if this would be acceptable (basically that we have to know the endpoint id for a channel)

Reply:
Yeah, I think that's reasonable for the new bridge, and the error explains what's going on.